safe-global/web-core
# style: Adjust tx summary columns to have fixed width

## What it solves

Resolves #675 #682 

## How this PR fixes it

Adds a min-width to the dynamic columns and breaks the content into multiple lines if there is not enough space. This change comes with the trade-off that there will be more white space if a column has very little content e.g. a short method name.

## How to test it

1. Open the Safe
2. Navigate to the transaction history
3. Adjust the viewport width
4. Observe each transaction column to be on the same line
5. Observe columns with a lot of content breaking into multiple lines if necessary

## Screenshots
<img width="821" alt="Screenshot 2022-09-27 at 11 52 52" src="https://user-images.githubusercontent.com/5880855/192495707-219e7397-a214-45b5-a00f-3f402e61d209.png">
